packages/expo-image/src/web/ColorTintFilter.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { StyleSheet } from 'react-native';
+
+export function getColorTintStyle(tintColor?: string | null) {
+  if (!tintColor) return {};
+  return {
+    filter: `url(#tint-${tintColor})`,
+  };
+}
+
+export default function ColorTintFilter({ tintColor }: { tintColor?: string | null }) {
```

Comment:
`{ tintColor?: string | null }` -> `type ColorTintFilterProps`

tbh I would rename `ColorTint` to `TintColor` as well